WeekTwo

```diff
@@ -0,0 +1,68 @@
+/*Test cases
+ "abcabcbb": length 3
+Given "abcabcbb", the answer is "abc", which the length is 3.
+Given "bbbbb", the answer is "b", with the length of 1.
+Given "pwwkew", the answer is "wke", with the length of 3.
+Given "" : return 0
+
+Note that the answer must be a substring, "pwke" is a subsequence and not a substring.
+
+*/
+
+public int lengthOfLongestSubstring(String s) {
+
+        if(s==null)
+
+            return 0;
+
+                boolean[] flag = new boolean[256];
+
+                int result = 0;
+
+                int start = 0;
+
+                char[] arr = s.toCharArray();
```

Comment:
nit: Try to pick more descriptive variable names (ideally, your interviewer will be able to know what information a variable holds just by reading the name).